src/components/kpi/kpi.less

```diff
@@ -154,13 +154,30 @@
       text-align: center;
     }
 
-    .im-button.alert-caption {
+    .alert-caption {
       text-align: center;
       font-size: 11px;
-      width: 81%;
+      background: lighten(@kpi-negative-alert-label-background-color, 20%);
+      border-radius: 3px;
+      width: 100%;
       margin: 0 auto 5px auto;
       box-shadow: none;
       padding: 2px;
+      color: white;
+    }
+
+    .im-button.alert-config-btn {
```

Comment:
@adamaziz15 the `.im-button` class is for creating a impac-material square button. For the rounded buttons, you can utilise `.im-fab` (which will take care of the rounding you're doing). Take example from the remove button. The alert button should be `<button class="kpi-alerts im-fab im-primary">`. Then add your custom less styles above the kpi-remove button's styles in this file, customising the positioning. Also, I think the button style theme is to be black, and then the colour (blue in this instance) on hover. If implemented alike the remove button, it should work on mobile.